.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+node_js:
+- "0.10"
+before_install:
+- npm install -g grunt-cli bower
+- bower install
+script: grunt quality
```

Comment:
I'd rather just have this being part of `npm test`. Not only for CI just for development it's easy to remember if the command is the same everywhere.